dotCMS/src/main/java/com/dotcms/filters/interceptor/saml/SamlWebInterceptor.java

```diff
@@ -183,6 +198,12 @@ public Result intercept(final HttpServletRequest request,
         return Result.NEXT;
     } // intercept.
 
+    private HttpSession getSession (final HttpServletRequest httpServletRequest) {
+
+        final HttpSession session = httpServletRequest.getSession(false);
+        return session != null? session: httpServletRequest.getSession(true);
+    }
+
```

Comment:
I don't understand the need for this code?

`request.getSession()` does the same thing?